invenio_records_marc21/templates/invenio_records_marc21/landing_page/record.html

```diff
@@ -91,4 +91,4 @@
 {%- block javascript %}
 {{ super() }}
 {{ webpack["invenio-records-marc21-landing-page.js"] }}
-{%- endblock javascript %}
+{%- endblock javascript %}
```

Comment:
ccould you please remove this change from the commit. then it is ready and i will merge and release it